### PR TITLE
Added Hiring page

### PR DESCRIPTION
**Pull request by Shreya-IITJ**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -43,6 +43,7 @@
                                             <li><a href="index.html">Home</a></li>
                                             <li><a href="about.html">About</a></li>
                                             <li><a href="contact.html">Contact</a></li>
+                                            <li><a href="Hiring.html">Work With Us</a></li>
```
Comment (harsh-not-haarsh) [inline about.html:46]: follow the naming conventions, it should be `hiring.html` and not `Hiring.html`

File: about.html [hunk 2]
```diff
@@ -159,9 +160,10 @@ <h5>Our Mission</h5>
                                     <div class="footer-tittle">
                                         <h4>Navigation</h4>
                                         <ul>
-                                            <li><a href="/">Home</a></li>
-                                            <li><a href="/about.html">About</a></li>
-                                            <li><a href="/contact.html">Contacts</a></li>
+                                            <li><a href="index.html">Home</a></li>
+                                            <li><a href="about.html">About</a></li>
+                                            <li><a href="contact.html">Contacts</a></li>
+                                            <li><a href="Hiring.html">Work With Us</a></li>
```
Comment (harsh-not-haarsh) [inline about.html:166]: follow the naming conventions, it should be `hiring.html` and not `Hiring.html`

File: Hiring.html [hunk 3]
```diff
@@ -0,0 +1,309 @@
+<!doctype html>
```
Comment (harsh-not-haarsh) [inline Hiring.html:1]: add relevant screenshots of this page in the PR

**Review by harsh-not-haarsh — CHANGES_REQUESTED** [review]

**Comment by Shreya-IITJ** [comment]
ARED - Personal - Microsoft​ Edge 26-02-2021 17_05_59
ARED - Personal - Microsoft​ Edge 26-02-2021 17_06_22
ARED - Personal - Microsoft​ Edge 26-02-2021 17_06_31

**Comment by harsh-not-haarsh** [comment]
three more things are required here:

- [ ] Rename the qualifications field to "Why do you want to join us?"
- [ ] Write a line on top "Please fill the form, we will contact you if we have an empty position that suits you"
- [ ] Display a confirmation message on successful form submission. (Something like "Thanks! We will contact you if there is a suitable position")

**Pull request merged**